admin/app/components/solidus_admin/products/index/component.html.erb

```diff
@@ -27,17 +27,10 @@
         icon: 'play-circle-line',
       },
     ],
-    columns: [
-      {
-        class_name: "w-10",
-        header: tag.span('aria-label': t('.product_image'), role: 'text'),
-        data: -> { image_column(_1) },
-      },
-      { header: :name, data: -> { name_column(_1) } },
-      { header: :status, data: -> { status_column(_1) } },
-      { header: :price, data: -> { price_column(_1) } },
-      { header: :stock, data: -> { stock_column(_1) } }
-     ]
+    columns: @columns,
+    column_classes: {
+      image: "w-10"
```

Comment:
I'm sure this is intentional but what's the reasoning behind not having the classes in the Column itself? It's true that this file might be responsible for styling and the data layer shouldn't know about it, but might have a specific style depending on the data of the column. Seeing it from this angle, I probably see this class more close to the column definition. If people add a new column, where should they define their class?

Reply:
Yeah, that also deserves more attention. It can probably be added to the column definition.